core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -1713,14 +1665,13 @@ oppia.factory('computeGraphService', [
       var nodes = {};
       var links = [];
       var finalStateIds = [];
-      for (var stateName in states) {
-        var interaction = states[stateName].interaction;
+      for (var stateName in states.getStates()) {
```

Comment:
states.getStateNames(); ditto below.

That said -- maybe it makes sense to encapsulate some of these checks within the States object? I.e. States.getFinalStateNames(), States.getNullInteractionStateNames(), etc.